contracts/tests/sbtc-deposit.test.ts

```diff
@@ -63,11 +76,13 @@ describe("sBTC deposit contract", () => {
         topic: string;
         txid: string;
         voutIndex: bigint;
+        amount: bigint;
       }>(print.data.value);
       expect(printData).toStrictEqual({
         topic: "completed-deposit",
         txid: new Uint8Array(32).fill(0),
         voutIndex: 0n,
+        amount: 1000n,
       });
```

Comment:
Can you add a test for the ft-mint event, and also that alice's balance is correct?

You can check ft events with:

```ts
const transferEvents = filterEvents(
  receipt.events,
  CoreNodeEventType.FtMintEvent
);
expect(transferEvents.length).toEqual(1);
const [transfer] = transferEvents;
expect(transfer.data).toStrictEqual({
  asset_identifier: `${token.identifier}::${token.fungible_tokens[0].name}`,
  sender: alice,
  amount: 1000n,
});
```

Reply:
Oh, you have the balance check in the token tests 😄